opencti-platform/opencti-graphql/src/database/file-storage.js

```diff
@@ -147,7 +147,7 @@ export const storeFileConverter = (user, file) => {
   };
 };
 
-export const loadFile = async (context, user, filename) => {
+export const loadFile = async (_, user, filename) => {
```

Comment:
Can't we completely removed the argument like for `downloadFile` ?

Reply:
Of course, I will do it